src/components/SignUpModal.js

```diff
@@ -0,0 +1,43 @@
+import { CancelSignInIcon } from '../icons'
+import React from 'react'
+import '../index.css'
+
+const SignUpModal = ({setShowSignUpModal}) => {
+  return (
+    <>
+    <div className='signInModal'>
+        <div className='sign-in-container'>
+        <form className='signInContent'>
+        <div className='signInHeader'>
+                    <h2>Sign Up</h2>
+                    <div className='CancelIcon' onClick={() => setShowSignUpModal(false)}>
+                        <CancelSignInIcon />
+                    </div>
+                </div>
+                <div>
+                    <input className='signInEmail' type="text" placeholder='First and last name'></input>
```

Comment:
Non-blocking: usually, its a good idea to have your input elements and even buttons as components so they can be reused and consistent in behaviour

Reply:
![Screenshot (138)](https://user-images.githubusercontent.com/73981407/221431111-748677a7-22f8-417d-a60c-87ee10bc81bb.png)

Reply:
![Screenshot (139)](https://user-images.githubusercontent.com/73981407/221431131-638c38b6-23fd-4b44-8dd0-1a9be0beca76.png)